src/project/projecttypes.h

```diff
@@ -169,6 +169,7 @@ struct ProjectMeta
     {
         bool equal = filePath == other.filePath;
         equal &= subtitle == other.subtitle;
+        equal &= title == other.title;
```

Comment:
I think it would be great to keep the struct members in the same order as they are declared (just to avoid confusion in the future). So could you please put the line about `title` above `subtitle`?

Reply:
Should I also move the thumbnail before the creation date?

Reply:
Nah, let's leave that one at the bottom, since it's a special case.

Reply:
It is unusual, but possible, for a **score** to have a _partName_ too, shouldn't that get checked too?

Reply:
If there is a partName, that will be part of `additionalTags`, so it is indeed checked too.

Reply:
Ah, I see